CMakeLists.txt

```
@@ -15,3 +20,6 @@ message(
     "\nThis script will build third party components licensed under various open source licenses into your container images.\n"
     "The terms under which those components may be used and distributed can be found with the license document that is provided with those components.\n"
     "Please familiarize yourself with those terms to ensure your distribution of those components complies with the terms of those licenses.\n\n")
```

```suggestion
    "Please familiarize yourself with those terms to ensure your distribution of those components complies with the terms of those licenses.\n\n"
    "\n-- Setting: SECURITY_STRICT=${SECURITY_STRICT}\n")
```

Just append it to here

---

Updated. Thanks